ide-configuration/eclipse.importorder

```diff
@@ -0,0 +1,7 @@
+#Organize Import Order
+#Wed Jan 23 12:03:29 AEDT 2019
+0=java
+1=javax
+2=org
+3=com
```

Comment:
What about packages starting with something else, like ch, it, cz, io, etc.?

Reply:
Packages that are not mentioned here go after the mentioned ones and are sorted lexicographically.

Reply:
Thanks!